botocore/args.py

```diff
@@ -84,6 +85,8 @@ def get_client_args(
         scoped_config,
         client_config,
         endpoint_bridge,
+        endpoints_ruleset_data,
+        partition_data,
         auth_token=None,
     ):
```

Comment:
I'm a little worried about changing the function signature like this. It looks like we're already accounting for endpoints_ruleset_data being `None` below. Can we do the same for partitions and place both of these at the end of the function signature?

```suggestion
        client_config,
        endpoint_bridge,
        auth_token=None,
        endpoints_ruleset_data=None,
        partition_data=None,
    ):
```